src/transformers/tokenization_utils.py

```diff
@@ -410,7 +410,7 @@ def __len__(self):
         Size of the full vocabulary with the added tokens. Counts the `keys` and not the `values` because otherwise if
         there is a hole in the vocab, we will add tokenizers at a wrong index.
         """
-        return len(set(self.get_vocab().keys()))
+        return max(set(self.get_vocab().keys()))
```

Comment:
This is done to match the `fast` version. Can be removed if it causes other tests to fail!